src/pages/mod-guidelines.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Rokucraft Rules
+layout: "../layouts/MarkdownPage.astro"
+---
+
+# Mod Guidelines
+
+The changes different mods make to your game can vary wildly, so these guidelines aim to help you understand what mods would or would not be in violation of our rules.
```

Comment:
```suggestion
The changes different mods make to your game can vary wildly, so these guidelines aim to help you understand which mods would or would not be in violation of our rules.
```

I could be wrong and missing some rule of the English language that I don't know, but in my brain it feels like this should say 'which' rather than 'what'